packages/CodingStandard/src/Fixer/DependencyInjection/InjectToConstructorInjectionFixer.php

```diff
@@ -0,0 +1,239 @@
+<?php declare(strict_types=1);
```

Comment:
@SpacePossum I'm proud to inform you I have added 2nd fixer, a bit more advanced then the first one.

Could I humbly ask your for a review and how to improve it?

The most difficult issue here:

- I scan for properties
- I add the first to constructor, create it if is missing
- **I add the second to constructor, that was added by first proprerty**

I managed to do this by rerun (something like "goto") checker over updated set of tokens.
The line is right here: https://github.com/Symplify/Symplify/pull/236/files#diff-0c7942472a912c2c38e42b9bc5edb5c7R108

Is there a way to do this better? Thanks a lot!